### PR TITLE
Fixes #13338 : UI Automation: Write an automated test using Cypress for "Long lists of selections are scrollable "

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Summary
Added test to check if the list is scrollable when its huge inside webhook message with menu attachment.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/13338
Jira https://mattermost.atlassian.net/browse/MM-21035

### CODE REVIEW

File: e2e/cypress/fixtures/message-attachments-with-menu-options.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
+{
```
Comment (saturninoabril) [inline e2e/cypress/fixtures/message-attachments-with-menu-options.json]: In filename, please change dash (`-`) to underscore (`_`).

File: e2e/cypress/fixtures/message-attachments-with-menu-options.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    "simple-list": [{
```
Comment (saturninoabril) [inline e2e/cypress/fixtures/message-attachments-with-menu-options.json]: `simple-list` is not used elsewhere.  Do you have plan to replace the existing ones?

File: e2e/cypress/integration/interactive_menu/basic_options_spec.js [hunk 3]
```diff
@@ -270,6 +271,56 @@ describe('Interactive Menu', () => {
...
+                    // # But the top options should be visible
+                    cy.findByText(longListOptions[0].text, {listContainer}).should('be.visible');
+                    cy.findByText(longListOptions[1].text, {listContainer}).should('be.visible');
+                });
+            });
+
+            // # Close message attachment menu dropdown
+            cy.get('body').click();
+        });
+    });
```
Comment (saturninoabril) [inline e2e/cypress/integration/interactive_menu/basic_options_spec.js]: 💯

File: e2e/cypress/fixtures/message-attachments-with-menu-options.json (hunk 2 above)
Comment (M-ZubairAhmed) [reply e2e/cypress/fixtures/message-attachments-with-menu-options.json]: I have removed this for now,
After all the PRs of mine are merged i plan to change simpleoptions from this file instead

**Comment by M-ZubairAhmed** [comment]
@srkgupta fixed the conflicts. Thank you

**Pull request merged**